.jules/runs/bridge_bindings_wasm/decision.md

```
@@ -0,0 +1,8 @@
+# Option A: Extend web/runner to support in-memory `base64` inputs
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)